ta/storage/storage.c

```diff
@@ -485,3 +485,62 @@ TEE_Result ta_storage_cmd_reset_obj(uint32_t param_types, TEE_Param params[4])
 	TEE_ResetTransientObject(o);
 	return TEE_SUCCESS;
 }
+
+TEE_Result ta_storage_cmd_info_in_persistent(uint32_t param_types,
+					    TEE_Param params[4])
+{
+	TEE_Result res;
+	TEE_ObjectHandle object;
+	TEE_ObjectInfo info1;
+	TEE_ObjectInfo info2;
+	char write_data[50];
```

Comment:
minor: magic "50" for data length, maybe it's better to have one constant somewhere for it?